aws_lambda_builders/workflows/java_gradle/workflow.py

```diff
@@ -0,0 +1,83 @@
+"""
+Java Gradle Workflow
+"""
+import hashlib
+import os
+from aws_lambda_builders.path_resolver import PathResolver
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+from .actions import JavaGradleBuildAction, JavaGradleCopyArtifactsAction
+from .gradle import SubprocessGradle
+from .utils import OSUtils
+from .gradlew_resolver import GradlewResolver
+from .gradle_validator import GradleBinaryValidator
+
+
+class JavaGradleWorkflow(BaseWorkflow):
+    """
+    A Lambda builder workflow that knows how to build Java projects using Gradle.
+    """
+    NAME = "JavaGradleWorkflow"
+
+    CAPABILITY = Capability(language="java",
+                            dependency_manager="gradle",
+                            application_framework=None)
+
+    INIT_FILE = "lambda-build-init.gradle"
+
+    def __init__(self,
+                 source_dir,
+                 artifacts_dir,
+                 scratch_dir,
+                 manifest_path,
```

Comment:
You should also verify that this manifest file exists and fail the workflow if it does not. We don't validate this in SAM CLI side because the builder is the authority on manifests.

Reply:
Similar to this https://github.com/awslabs/aws-lambda-builders/blob/develop/aws_lambda_builders/workflows/python_pip/packager.py#L37-L40

Reply:
+1